src/components/actionSheet/index.js

```diff
@@ -59,6 +59,27 @@ export default class ActionSheet extends BaseComponent {
      * When passed (only with useNativeIOS), will display a cancel button at the bottom (overrides cancelButtonIndex)
      */
     showCancelButton: PropTypes.bool,
+    /**
+     * Add or override style of the action sheet
+     */
+    sheetStyle: ViewPropTypes.style,
+    /**
+     * Add or override style of the dialog wrapping the action sheet
+     */
+    dialogStyle: ViewPropTypes.style,
+    /**
+     * Add or override style of the options list
+     */
+    listStyle: ViewPropTypes.style,
```

Comment:
is listStyle is action/optionStyle?

Reply:
It is the style of all the actions, not a single one, do you like actionsListStyle more?

Reply:
I think using the word 'list' here is confusing (even if it eventually renders List item). We are referring to these as 'options' and 'actions' so we should stick to one and not add another one to the mix.

Reply:
Done